packages/react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -244,9 +244,10 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef<
       <div role="toolbar" {...divProps} ref={ref}>
         <FocusZone
           componentRef={focusZoneRef}
-          direction={FocusZoneDirection.horizontal}
           className={classNames.root}
```

Comment:
You should move this after you spread the props and do:

```suggestion
          className={css(classNames.root)}
```

where `css` is imported from `@fluentui/utilities`. You should do this so that styles are not lost if someone passes another `className` as part of `focusZoneProps`.

Reply:
Sorry, my bad here, I gave the wrong answer, you should move this to after the spread and do this:

`className={css(classNames.root, focusZoneProps.className)}`